package.json

```diff
@@ -75,7 +75,7 @@
     "pod-install": "^0.1.0",
     "prettier": "^2.0.5",
     "react": "18.2.0",
-    "react-native": "0.73.5",
+    "react-native": "^0.73.5",
```

Comment:
Is this needed for the fix? Though I think it should be ok 👍

Reply:
Not needed by I added it just in case they had released a patch on that version, which they hadn't. So I just left it just in case 🤷‍♂️